Tests/js/models/assets/ez-locationmodel-tests.js

```diff
@@ -504,10 +504,132 @@ YUI.add('ez-locationmodel-tests', function (Y) {
         }
     });
 
+    loadPathTest = new Y.Test.Case({
+        name: "eZ location model load path tests",
+
+        setUp: function () {
+            this.capiMock = new Y.Mock();
+            this.contentService = new Y.Mock();
+            this.locationId = '/1/2/3/4';
+            this.model = new Y.eZ.Location({id: this.locationId});
+            this.loadAncestorsResponse = {
+                document: {
+                    View: {
+                        Result: {
+                            searchHits: {
+                                searchHit: [
+                                    {value: { Location: {_href: '/parent/location', depth: 2}}},
+                                    {value: { Location: {_href: '/grand/parent/location', depth: 1}}},
+                                    {value: { Location: {_href: this.locationId, depth: 3}}},
+                                    {value: { Location: {_href: '/home/location', depth: 0}}},
+                                ]
+                            }
+                        }
+                    }
+                }
+            };
+            this.viewCreateStruct = {
+                body: {
+                    ViewInput: {
+                        LocationQuery : {
+
+                        }
+                    }
+                }
+            };
+
+            Y.Mock.expect(this.capiMock, {
+                method: 'getContentService',
+                returns: this.contentService,
+            });
+
+            Y.Mock.expect(this.contentService, {
+                method: 'newViewCreateStruct',
+                args: [Y.Mock.Value.String, 'LocationQuery'],
+                returns: this.viewCreateStruct
+            });
+        },
+
+        tearDown: function () {
+            this.model.destroy();
+            delete this.model;
+        },
+
+        "Should load path for the location": function () {
+            var that = this, i;
+
+            Y.Mock.expect(this.contentService, {
+                method: 'createView',
+                args: [Y.Mock.Value.Object, Y.Mock.Value.Function],
+                run: function (options, callback) {
```

Comment:
`options` should be renamed to `query` and you should assert it contains a query with the ancestor criterion correctly configured.